portfolio/src/main/webapp/aboutMe.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>About Me | My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  </head>
+  <body>
+    <div id="content" class="flex-container">
+        <div class="header-div">
+            <h1 class="heading-text">About Me</h1>
+        </div>
+        <div class="flex-item">
+            <h2 class="subheading-text">Basic Info</h2>
+            <p class="body-text">
+                <b>Preferred Name:</b> Anna Giang <br>
+                <b>Preferred Pronoun:</b> She/her <br>
+                <b>Birthday</b> 12th November <br>
+                <b>Location:</b> Melbourne, Victoria, Australia
+            </p>
+        </div>
+        <div class="flex-item">
+            <img class="section-img" src="/images/covers/about-me-cover.jpg" alt="Anna is Shanghai">
+        </div>
+        <div class="flex-item">
+            <h2 class="subheading-text">Background</h2>
+            <p class="body-text">Aspiring software engineer. Penultimate year student in Bachelor of 
+            Software Engineering (Hons) and Bachelor of Arts (Chinese Studies 
+            and Korean Studies majors) at Monash University.
+            </p>
+        </div>
+        <div class="flex-item">
+            <h2 class="subheading-text">Experiences</h2>
+            <p class="body-text">STEP Intern at Google, 2020. Events Co-Director at Monash Association 
+            of Coding (MAC). Engagement Coordinator at Robogals Monash. Korean 
+            Language Facilitator for Monash Student Association. Previous Marketing
+            Intern at the Korean Society of Victoria.
+            </p>
+        </div>
+        <div class="flex-item">
+            <h2 class="subheading-text">Interests and hobbies</h2>
+            <ul class="body-text">
+                <li>Coding competitions</li>
+                <span class="footnote-text">Complete beginner at this.</span>
```

Comment:
nit: spans shouldn't be nested within `<ul>` directly: https://html.spec.whatwg.org/multipage/grouping-content.html#the-ul-element (only `<li>` elements should be in `<ul>`). You could try something like
```
<li> Coding Competitions
<br><span> Complete beginner at this.</span>
</li>
```

You can use a validator to pick up on things like this: https://validator.w3.org/ (honestly I would probably miss this myself without it).

Reply:
Fixed! Thanks for your suggestion!

Reply:
Thanks for changing this